docs/Using Fleet/CIS-Benchmarks.md

```diff
@@ -105,11 +109,7 @@ To implement automated remediation, you can install a separate agent such as Mun
 
 ### Manual vs. automated
```

Comment:
@rachaelshaw, just spoke to @sharon-fdm and we decided that it makes sense to remove the concept of "manual v. automated remediation" entirely. 

Why? It just adds extra noise.

We can remove this section and remove the prefix for the resolutions (ex. "Automated method:"): https://github.com/fleetdm/fleet/blob/main/ee/cis/macos-13/cis-policy-queries.yml#L2192

Maybe makes sense to do as part of a separate PR so we can get this one merged. Up to you.

Reply:
➕